qscaler/configuration/config.py

```diff
@@ -8,6 +8,6 @@ class Config:
     pulling_interval: int = field(default=os.getenv("PULLING_INTERVAL", 2))
     pod_name: str = field(default=os.getenv("HOSTNAME"))  # in K8s it is the name of the pod
     pod_spec_hash: str = field(default=os.getenv("POD_SPEC_HASH"))
-
+    kubeconfig: str = field(default=os.getenv("KUBECONFIG"))
```

Comment:
where is that one used ? looks like load_config method searches for it by it self